campaignion_webform_widget/js/widget.js

```diff
@@ -51,14 +51,14 @@ function gaLinkerHandler() {
 
 window.addEventListener("DOMContentLoaded", function() {
   gaLinkerHandler();
-  messageParent();
+  messageParent(false);
 });
 
 window.onload = function() {
-  messageParent();
-  window.addEventListener("DOMSubtreeModified", messageParent, true);
+  messageParent(false);
+  window.addEventListener("DOMSubtreeModified", function() { messageParent(false); }, true);
```

Comment:
I’d find a new line easier to read and more consistent with the other event handlers.

```js
window.addEventListener("DOMSubtreeModified", function() {
  messageParent(false);
}, true);
```